### PR TITLE
Fix typo in progressindicator sample code

### PR DESCRIPTION
**Pull request by albertvaka**
Example for determinate progress bar had copy-and-paste errors.

### CODE REVIEW

**Comment by albertvaka** [comment]
@googlebot I signed it!

File: docs/components/ProgressIndicator.md [hunk 1]
```diff
@@ -245,8 +245,7 @@ In the layout:
 ```xml
 <com.google.android.material.progressindicator.LinearProgressIndicator
     android:layout_width="match_parent"
```
Comment (pekingme) [inline docs/components/ProgressIndicator.md:247]:
```suggestion
<com.google.android.material.progressindicator.CircularProgressIndicator
    android:layout_width="wrap_content"
```

**Review by pekingme — CHANGES_REQUESTED** [review]
Thanks for working on fixing the typo. The code snippet is a mistake copying from previous section. Could you also fix the suggested lines? Thanks.

File: docs/components/ProgressIndicator.md (hunk 1 above)
Comment (albertvaka) [reply docs/components/ProgressIndicator.md]: Sorry I didn't catch that earlier! Should be good now.

**Review by pekingme — APPROVED** [review]
LGTM, thanks.

**Comment by albertvaka** [comment]
I don't have superpowers to merge this, can somebody do it for me? 🙇

**Pull request closed without merging**